### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by ChadRehmKineticData**
The initial commit populated the master branch without a code review.  Could we get a full code review on the zoom api handler.  That is the only component in the repo at this time.

### CODE REVIEW

File: handlers/zoom_api_v1/readme.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
 **enable_debug_logging**: Yes or No. Controls logging behavior. 
 
 ## Parameters
-**Error Handling**
-  Select between returning an error message, or raising an exception.
-**Method**
+**Error Handling**: Select between returning an error message, or raising an exception.
+
+**Method**:
   HTTP Method to use for the Kinetic Core API call being made.
```
Comment (jackboespflug) [inline handlers/zoom_api_v1/readme.md]: This references the Kinetic Core API - should be changed to the Zoom API.

File: handlers/zoom_api_v1/readme.md [hunk 2]
```diff
@@ -18,11 +20,9 @@ Used for requests to the Google Cloud Apps REST API Client
   - PATCH
   - DELETE
 
-**Path**
-  The relative API path (to the `https://api.zoom.us/v2` info value) that will be called.
-  This value should begin with a forward slash `/`.  A list of [zoom](https://marketplace.zoom.us/docs/api-reference/zoom-api) API paths. 
-**Body**
-  The body content (JSON) that will be sent for POST, PUT, and PATCH requests.
+**Path** :The relative API path (to the `https://api.zoom.us/v2` info value) that will be called. This value should begin with a forward slash `/`.  A list of [zoom](https://marketplace.zoom.us/docs/api-reference/zoom-api) API paths.
```
Comment (jackboespflug) [inline handlers/zoom_api_v1/readme.md]: You reference "info value" here, but it appears the **Path** parameter is always relative to 'https://api.zoom.us/v2'.

**Review by jackboespflug — CHANGES_REQUESTED** [review]
The description in the node.xml file indicates this is a generic Rest handler, when it should be updated to reflect it is a Zoom API handler.

Update that and the other two items I commented on, and it will be good to go.

**Comment by ChadRehmKineticData** [comment]
@jackboespflug I made the requested changes.

**Review by jackboespflug — APPROVED** [review]
Looks good.

**Pull request merged**